src/vs/workbench/contrib/terminal/browser/terminalWidgetManager.ts

```diff
@@ -43,13 +43,13 @@ export class TerminalWidgetManager implements IDisposable {
 		mutationObserver.observe(this._xtermViewport, { attributes: true, attributeFilter: ['style'] });
 	}
 
-	public showMessage(left: number, top: number, text: string): void {
+	public showMessage(left: number, bottom: number, text: string, positionIsTopOfWidget: boolean = false): void {
```

Comment:
This could be a little nicer is `bottom` was renamed to `y` and we used something like this:

```ts
enum VerticalAlignment {
  Bottom,
  Top
}
```

Reply:
Is using `y` still clear that this is a bottom up position, as opposed to top down?